diff --git a/CMakeLists.txt b/CMakeLists.txt
index f0615027a7..869f54afd2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -18,7 +18,7 @@
 
 # We specify a minimum requirement of 3.10.2, but for now use 3.5.1 here
 # until our infrastructure catches up.
-cmake_minimum_required(VERSION 3.5.1)
+cmake_minimum_required(VERSION 3.5.1...3.10.2)
 
 include(CMakePackageConfigHelpers)
 
@@ -326,7 +326,8 @@ add_subdirectory(framework)

 add_subdirectory(include)

-add_subdirectory(3rdparty)
+# Exclude 3rdparty components (everest, p256-m) that are not needed
+# add_subdirectory(3rdparty)

 add_subdirectory(library)
